src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -299,15 +299,6 @@ summary {
 
 /* Fix some style changes due to normalize.css 8 */
 
-td,
-th {
-	padding: 0;
```

Comment:
I wonder if we relied on it for tables in docblock too? Well, that would be surprising to need it for tables in the first place.